frontend/public/components/utils/name-value-editor.jsx

```diff
@@ -518,15 +518,16 @@ const PairElement_ = DragSource(
                 </div>
               )}
               <div className="col-xs-5 pairs-list__name-field">
-                <input
-                  type="text"
-                  data-test="pairs-list-name"
-                  className="pf-v5-c-form-control"
-                  placeholder={nameString}
-                  value={pair[NameValueEditorPair.Name]}
-                  onChange={this._onChangeName}
-                  disabled={readOnly}
-                />
+                <span className="pf-v6-c-form-control">
```

Comment:
Heads up @sg00dwin since you're also working on updating form elements that use pf styling directly.

Reply:
I'm okay with reverting this commit, but this change shouldn't cause issues. It follows the markup provided by the [Patternfly HTML/CSS examples](https://www.patternfly.org/components/forms/form/html), so it removes the need for CSS overrides for these text boxes

Reply:
Yeah.  I am of the mindset though that it's better to be consistently wrong since this example is one of many that predates PatternFly.